database/seed.js

```diff
@@ -17,16 +17,17 @@ const generateRoomsData = () => {
       average_review_point: (Math.random() * (5 - 1) + 1).toFixed(2),
       number_of_reviews: Math.floor(Math.random() * (250 - 1) + 1),
       is_superhost: faker.random.boolean(),
+      is_favorite: Math.floor(Math.random() * (2 - 0) + 0),
```

Comment:
@tegsheeb Just out of curiosity, why did you choose to go with 0 vs 1 instead of true vs false with is_favorite?

Reply:
in MySQL, boolean is stored as 0 and 1s. I tried with boolean, but somehow it was not functioning(strange because  is_superhost is boolean). So, since true and false are stored as 0 and 1s in Mysql, tested with simpler version and it worked.